migrations/0012_separate_bet_token_value.up.sql

```diff
@@ -0,0 +1,16 @@
+ALTER TABLE game_sessions
+  ADD COLUMN symbol VARCHAR(10) DEFAULT '4,BET',
+  ADD COLUMN deposit_value NUMERIC DEFAULT 0,
+  ADD COLUMN player_win_value NUMERIC DEFAULT NULL,
+  ADD COLUMN token VARCHAR(7) DEFAULT 'BET';
```

Comment:
do u think it's better to have symbol or token and precision separately?

Reply:
it's better for memory optimization